.github/workflows/unittests.yml

```diff
@@ -1,4 +1,11 @@
-on: [push, pull_request]
+on: 
+  push:
+    paths: ['**.R', '**.svg', '**.Rd', 'DESCRIPTION', 'NAMESPACE']
```

Comment:
What about `cpp`, `h` and `Makevars`? And do we need `.Rd`?

Reply:
I added `'**.c', '**.cpp', '**.h', '**.hpp', 'MAKEVARS', 'MAKEVARS.win'`.

I prefer to leave `**.Rd` because the R CMD check (which I hope we will use one day into the future) does check the help files (all arguments of exported functions must be documented). We could omit it for now, but then we're probably gonna add it later anyway.

Also, the unit tests won't trigger if people adjust a data set for the unit tests, so this may need to be modified under certain circumstances anyway.

Reply:
We could also include `tests/**` because changes in the test folder should probably trigger the tests to run

Reply:
sure, that's probably better than `'**.svg'`